<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CloudItemWidget</class>
 <widget class="QWidget" name="CloudItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="moreButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&lt;more&gt; </string>
     </property>
     <property name="popupMode">
      <enum>QToolButton::InstantPopup</enum>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
     <property name="arrowType">
      <enum>Qt::NoArrow</enum>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionHideCloudPoints">
   <property name="text">
    <string>HideCloudPoints</string>
   </property>
  </action>
  <action name="actionShowCloudPoints">
   <property name="text">
    <string>ShowCloudPoints</string>
   </property>
  </action>
  <action name="actionHideCloudNormals">
   <property name="text">
    <string>HideCloudNormals</string>
   </property>
  </action>
  <action name="actionShowCloudNormals">
   <property name="text">
    <string>ShowCloudNormals</string>
   </property>
  </action>
  <action name="actionDeleteItem">
   <property name="text">
    <string>DeleteItem</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
